src/data/form-interaction.js

```diff
@@ -10,7 +10,7 @@ attachInputHandler();
 attachEmptyCellHandler();
 
 function attachInputHandler() {
-  table.addEventListener('input', Utils.throttle((e) => onThrottledInput(e, updateModel), 2000));
+  table.addEventListener('input', Utils.throttle((e) => onThrottledInput(e, updateModel), 800));
```

Comment:
Since I usually have a bunch of modifications to make on my PTO form, having to wait 2 seconds every time was a pain. I'm proposing we lower the delay to 800ms, I believe that's a fairly good number according to a recent STDDMC* survey. 

_* Sorti Tout Droit De Mon Cul_

Reply:
I love it, it's just as arbitrary as the previous value, so it can't be less good !